service/src/main/java/uk/gov/hmcts/reform/fpl/service/orders/generator/C43ChildArrangementOrderDocumentParameterGenerator.java

```diff
@@ -0,0 +1,134 @@
+        + "named in the Child Arrangements Order as a person with whom the child shall live (Sections13(1), (2) and"
+        + " (4) Children Act 1989). \n\n"
+        + "It may be a criminal offence under the Child Abduction Act 1984 to remove the child "
+        + "from the United Kingdom without the leave of the court. \n\n"
+        + "Where a Child Arrangements Order is in force: if "
+        + "you do not comply with a provision of this Child Arrangements Order– \n\n"
+        + "(a) you may be held in contempt of court and be committed to prison or fined; and/or \n"
+        + "(b) the Court may make an order requiring you to undertake unpaid work (\"an "
+        + "enforcement order\") and/or an order that you pay financial compensation.";
+
+    public static final String WHERE = "Where - \n"
+        + "(a) there are no proceedings pending under Part 2 Children Act 1989; \n"
+        + "(b) an officer of the service/ Welsh family proceedings officer who remains involved "
+        + "with the case is given cause to suspect, whilst this order is in force, that the "
+        + "child concerned is at risk of harm; and \n"
+        + "(c) as a result that officer makes a risk assessment under section 16A of that Act, "
+        + "the officer may apply to the court for it to revive the previous proceedings and "
+        + "to consider that risk assessment and give such directions as the court thinks "
+        + "necessary.";
+
+    public static final String NOTICE = "Any person with parental responsibility for the child may obtain "
+        + "advice on what can be done to prevent the issue of a passport to the child. They should write to London "
+        + "Passport Office, Globe House, 89 Eccleston Square, LONDON SW1V 1PN.";
+
+    private final LocalAuthorityNameLookupConfiguration laNameLookup;
+
+    @Override
+    public Order accept() {
+        return Order.C43_CHILD_ARRANGEMENT_SPECIFIC_ISSUE_PROHIBITED_STEPS_ORDER;
+    }
+
+    @Override
+    public DocmosisParameters generate(CaseData caseData) {
+        ManageOrdersEventData eventData = caseData.getManageOrdersEventData();
+
+        String localAuthorityCode = caseData.getCaseLocalAuthority();
+        String localAuthorityName = laNameLookup.getLocalAuthorityName(localAuthorityCode);
+
+        C43ChildArrangementOrderDocmosisParameters.C43ChildArrangementOrderDocmosisParametersBuilder<?, ?>
+            c43DocmosisParameters = C43ChildArrangementOrderDocmosisParameters
+            .builder()
+            .orderTitle(getOrderTitle(eventData))
+            .orderByConsent(orderMessageGenerator.getOrderByConsentMessage(eventData))
+            .orderDetails(getOrderRecitalsAndPreambles(eventData))
+            .furtherDirections(getOrderDirections(eventData))
+            .localAuthorityName(localAuthorityName)
+            .noticeHeader("Notice")
+            .noticeMessage(NOTICE);
```

Comment:
You can see here that the variable name "NOTICE" clashes with "Notice" and may make it difficult to understand